project/config/mentalhealthchampionni/config/search_api.server.solr_default.yml

```diff
@@ -40,7 +40,7 @@ backend_config:
   connector: standard
   connector_config:
     scheme: http
-    host: mentalhealthchampionni_solr
+    host: mentalhealthchampion_solr
```

Comment:
This should be the original mentalhealthchampionni_solr value

Reply:
Wow, that's a good spot !